src/test/asmtext_filter_tests.cpp

```diff
@@ -76,6 +76,36 @@ TEST_CASE("gcc12_bin_fmt_O2_flto")
     ApprovalTests::Approvals::verify(ss.str());
 }
 
+TEST_CASE("gcc12_sort_object_reloc")
+{
+    AsmParser::Filter filter;
+    filter.binary = true;
+    filter.plt = false;
+    filter.library_functions = false;
+    filter.unused_labels = false;
+
+    std::string asmpath;
+    if (std::filesystem::current_path().string().ends_with("test"))
+    {
+        asmpath = "../../../resources/gcc12_sort_object_reloc";
```

Comment:
missing.asm

Reply:
oh right, my local test must have used the else part.